content/hardware/02.uno/boards/uno-q/tutorials/05.ssh/ssh.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Connect to UNO Q via Secure Shell (SSH)
+description: Learn how to access an UNO Q's shell via SSH.
+tags: [UNO Q, SSH, Terminal]
+author: Karl Söderby
+---
+
+The [Arduino® UNO Q](https://store.arduino.cc/products/uno-q) can be programmed via the [Arduino App Lab](/software/app-lab/), a desktop application supporting code editing and running [Apps](/software/app-lab/tutorials/getting-started/#create--run-apps) on the board without any external tools.
+
+The board's microprocessor (running Debian OS) can also be accessed using Secure Shell (SSH), a method that allows remote connection to your board over a local network. This allows you to, among other things:
+- Access the board's shell and perform operations on the board remotely.
+- Transfer files from your local computer to the board remotely (using SCP).
+
+The UNO Q board comes pre-packaged with the Arduino App Lab, which is based on the `arduino-app-cli` tool<sup>1</sup>. This allows you, among other things, to **launch Apps through the command line**, instead of using the desktop App.
+
+<sup>1</sup> Read more about the `arduino-app-cli` in the [Arduino App CLI guide](/software/app-lab/tutorials/cli/).
+
+## Requirements
+
+### Hardware Requirements
+
+- [Arduino® UNO Q](https://store.arduino.cc/products/uno-q)
+- 5 VDC 3 A power supply (e.g., a phone charger or USB port from a computer)
+
+### Software Requirements
+
+- [First setup](/software/app-lab/tutorials/getting-started/#install--set-up-arduino-app-lab) of board completed\*
```

Comment:
I would clarify either here or in the tutorial that the first configuration through the app-lab will enable by default network mode, which could also be enabled and disabled by the `arduino-app-cli` with 
```
arduino-app-cli system network-mode enable/disable
```